### ISSUE TITLE
Documentation on Pursuit

### ISSUE THREAD

**Issue by stevenyap** [nsaunders/purescript-tecton#20]
### Problem
I was trying to find the documentation of this package on [Pursuit](https://pursuit.purescript.org/) but it is not available.

### Requested enhancement
Please publish this package to Pursuit.

### References
NA

### Anything else?
If the package needs contributors for the documentation, maybe open an issue so that people can join in?
Thanks for the great work! It's a lot of effort to write this library.

**Comment by nsaunders** [comment]
Hi @stevenyap, thanks for your interest in Tecton, and apologies for the inconvenience. Previously, [an issue was encountered](https://github.com/purescript/registry/issues/32#issuecomment-1279612668) with the new automated publishing workflow that resulted in the docs not being uploaded to Pursuit. Now I've manually published the docs which can be found [here](https://pursuit.purescript.org/packages/purescript-tecton/0.1.6).

Admittedly, aside from the [test suite](./test) and [examples](./examples), the documentation is a bit sparse at this point and could definitely use some attention; and, indeed, contributions are welcome. I did start working on a [syntax guide](./docs/syntax.md), but I am not sure how useful the format is and would greatly appreciate any feedback and ideas you might have to share...
